10_deploy_rook.sh

```diff
@@ -13,13 +13,15 @@ oc create -f common.yaml
 sed '/FLEXVOLUME_DIR_PATH/!b;n;c\          value: "\/etc/kubernetes\/kubelet-plugins\/volume\/exec"' operator-openshift.yaml > operator-openshift-modified.yaml
 sed -i 's/# - name: FLEXVOLUME_DIR_PATH/- name: FLEXVOLUME_DIR_PATH/' operator-openshift-modified.yaml
 oc create -f operator-openshift-modified.yaml
+sleep 120
 oc wait --for condition=ready  pod -l app=rook-ceph-operator -n rook-ceph --timeout=120s
```

Comment:
I'm wondering why we need the sleep, when we already wait 120s for several conditions related to the operator/apps being ready below?